README.md

```diff
@@ -30,6 +30,8 @@ Your MySQL Data will be written to your local disk in the `data` folder to keep
 
 Your server should now be available under http://localhost:3000
 
+**Note**: When running the server locally it is by default ran in a Hot Reload Mode. This means that every time you change any of the source files locally the server is restarted to reflect the changes. This is in particular helpful when doing local development.
```

Comment:
Does this apply only if you start server via `yarn start:local`, or does it apply even with Docker as well?

Reply:
Should apply for Docker as well:

https://github.com/standardnotes/syncing-server-js/blob/68adf41b1adc36f42eadefbfedf751cc0590cc7c/docker-compose.yml#L6

https://github.com/standardnotes/syncing-server-js/blob/284e7e394060fe1ee0c8163960a8350591841662/docker/entrypoint.sh#L9

Reply:
Yes, this applies to Docker.

no point in starting `syncing-server-js` without Docker due to required services that it depends on